### PR TITLE
docs: add docs for contributors

### PR DESCRIPTION
**Pull request by ACHultman**
I want to add a style guide now to help y'all get more familiar with TS/JS and React

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+1. Search existing issues before submitting a new one.
+2. When creating a new issue, provide as much detail as possible (e.g., OS, environment, steps to reproduce).
+
+## Branching Strategy
+
+We use a branch naming convention to keep our repository organized. Follow the naming conventions below depending on the type of contribution you're making:
+
+### Main Branch
+
+- **main**: The stable branch used for production-ready code.
```
Comment (AhmedNSidd) [inline CONTRIBUTING.md:16]: Do you think we should have a `develop` branch and branch our feature branches from develop? It could be worth it...

**Review by AhmedNSidd — DISMISSED** [review]
Just one comment, but it LGTM. Thanks for the work 🙏

File: CONTRIBUTING.md (hunk 1 above)
Comment (ACHultman) [reply CONTRIBUTING.md:16]: I quit

Comment (ACHultman) [reply CONTRIBUTING.md:16]: jk

**Review by AhmedNSidd — APPROVED** [review]

**Review by JayKirsh — APPROVED** [review]
Code is clear!

**Pull request merged**